packages/structure/package.json

```diff
@@ -17,15 +17,15 @@
     "dotenv-defaults": "2.0.2",
     "enquirer": "2.3.6",
     "findup-sync": "4.0.0",
-    "graphql": "15.5.1",
+    "graphql": "15.5.3",
     "lazy-get-decorator": "2.2.0",
     "line-column": "1.0.2",
     "lodash": "4.17.21",
     "lodash-decorators": "6.0.1",
     "lru-cache": "6.0.0",
     "proxyquire": "2.1.3",
     "toml": "3.0.0",
-    "ts-morph": "11.0.3",
+    "ts-morph": "12.0.0",
```

Comment:
Do we need resolutions for ts-morph?

Reply:
Breaking here is ts-morph dependency upgrade to TS 4.4:
https://github.com/dsherret/ts-morph/blob/latest/packages/ts-morph/CHANGELOG.md#1200-2021-08-27